polclassifier/params.py

```diff
@@ -54,10 +53,8 @@
 LEAF_SIZE_DEFAULT = 4
 WEIGHTS_DEFAULT = 'distance'
 
-MODEL_TARGET = "local"
-
-BUCKET_NAME = "polclassifier-jonahramchandani"
-
```

Comment:
Please add these two back (otherwise the `save_model` functions I think will break without a `MODEL_TARGET` variable. The new stuff can stay!